modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -843,11 +813,17 @@ There are no valid index configuration values for the constraint-backing range i
 === Handling existing constraints when creating a constraint
 
 Creating an already existing constraint will fail.
+This includes the following scenarios:
+
+* Creating a constraint identical to an already existing constraint.
+* Creating a constraint with a different name but on the same schema as an already existing constraint.
+* Creating a constraint with the same name but on a different schema as an already existing constraint.
```

Comment:
```suggestion
* Creating a constraint with the same name as an already existing constraint, regardless of what that constraint is.
```
since it's not just the schema part (traditionally only the label/reltype and properties) that can differ. And I want to be clear that it covers literally only the name being the same.